### DISCUSSION TITLE
Add a Timing/Judgement Offset to Lazer

### DISCUSSION THREAD

**Discussion by FemboySkitty** [ppy/osu#20365] [Ideas]
One thing that I noticed about Lazer is that there is only audio/visual offset and not timing offset. This is unlike regular Osu's global offset where it actually affects when notes time; by changing the global offset in regular Osu you can shift where notes hit on the error bar. I think Lazer should also have a similar feature added because as great as the UI, customization, and new features are having every other hit register as an early 100 gets annoying.

**Comment by peppy** [comment]
Offset implementation is the same as in stable. You can use universal offset as you were. I think you may be misunderstanding something.

**Comment by FemboySkitty** [comment]
Really? Thank you for letting me know. Perhaps I was just not setting the right values.
